Makefile

```diff
@@ -453,6 +493,8 @@ pkg-windows: check-pkg-target-windows $(STATUS_CLIENT_EXE)
 
 zip-windows: check-pkg-target-windows $(STATUS_CLIENT_7Z)
 
+deb: check-pkg-target-linux $(STATUS_CLIENT_DEB)
```

Comment:
Shouldn't this be `deb-linux`? Or really, the targets should start with the platform, so it's:
```
make linux-deb
make linux-app
make windows-zip
make windows-exe
```